### PR TITLE
Added Update Feedback form Functionality 

### PR DESCRIPTION
**Pull request by Platoonhead**
-- Added Update Feedback form Functionality 
--Added test cases for feedback form update  associated functions in repository and controllers 
-- UI enhancement

### CODE REVIEW

File: app/controllers/FeedbackFormsController.scala [hunk 1]
```diff
@@ -131,4 +120,68 @@ class FeedbackFormsController @Inject()(val messagesApi: MessagesApi,
...
+    s"{${builder(feedForm.questions, Nil, 0).mkString(",")}}"
+  }
+
+  def updateFeedbackForm: Action[JsValue] = AdminAction.async(parse.json) { implicit request =>
+    request.body.validate[FeedbackFormInformation].asOpt.fold {
+      Logger.error(s"Received a bad request while updating feedback form, ${request.body}")
+      Future.successful(BadRequest("Malformed data!"))
+    } { feedbackFormInformation =>
+      val formValid =
+        feedbackFormInformation.validateForm orElse feedbackFormInformation.validateOptions orElse feedbackFormInformation.validateQuestion
```
Comment (teenasharma) [inline app/controllers/FeedbackFormsController.scala]: add validation for form name

File: app/models/FeedbackFormsRepository.scala [hunk 2]
```diff
@@ -45,6 +52,19 @@ class FeedbackFormsRepository @Inject()(reactiveMongoApi: ReactiveMongoApi) {
         jsonCollection
           .insert(feedbackData))
 
+  def update(id:String,feedbackData: FeedbackForm)(implicit ex: ExecutionContext): Future[WriteResult] = {
```
Comment (teenasharma) [inline app/models/FeedbackFormsRepository.scala]: code need to be reformate

File: app/controllers/FeedbackFormsController.scala (hunk 1 above)
Comment (Platoonhead) [inline app/controllers/FeedbackFormsController.scala]: Done

File: app/models/FeedbackFormsRepository.scala (hunk 2 above)
Comment (Platoonhead) [inline app/models/FeedbackFormsRepository.scala]: Done

**Pull request merged**